generator/config/search/embeddedDocument.yaml

```diff
@@ -0,0 +1,155 @@
+# $schema: ../schema.json
+name: embeddedDocument
+link: 'https://www.mongodb.com/docs/atlas/atlas-search/embeddedDocument/'
```

Comment:
This looks like a 404. Also, I need to reiterate how annoying it is that our docs website redirects 404s to https://www.mongodb.com/docs/404/ instead of leaving the original URL as-is. I've reported this in the past and the feedback wasn't acted upon :disappointed: 

I think this should be https://www.mongodb.com/docs/atlas/atlas-search/embedded-document/

Reply:
```suggestion
link: 'https://www.mongodb.com/docs/atlas/atlas-search/embedded-document/'
```